static/docs/user-guide/troubleshooting.md

```diff
@@ -0,0 +1,17 @@
+# Troubleshooting
+
+In this section we cover some of the known issues that DVC user might stumbe
+upon.
+
+## Too many open files error
+
+A known problem some users run into with `pull`, `fetch` and `push` commands is
```

Comment:
`dvc pull`, `dvc fetch`, `dvc push` - to make them links to the CR (command reference)

Reply:
```suggestion
A known problem some users run into with the `dvc pull`, `dvc fetch` and `dvc push` commands is
```
(Needs rewrap.)